### DISCUSSION TITLE
Rubrica Final Semana del Proyecto Grupo 8

### DISCUSSION THREAD

**Discussion by Gona79** [ISPC-TST-Electronica-Microcontrolada/Grupo8#82] [Announcements]
Nota de la 1era Semana: 8 (ocho)

Que tal estimados, trabajaron muy bien. 

a) Diagramaron el proyecto.
b) Lo perfilaron temporalmente con un diagrama gantt
c) Generaron la presentacion audiovisual del mismo
d) Explicaron la utilizacion del controlador
Aun asi, no completaron los formulario que subio el profe Jorge Morales @JorEl057. Por otra parte siempre es bueno pensar en la tecnologia como un recurso, para implementar una idea o control de proceso. Por ejemplo un robot asistente para personas no vidente, o un robot scanner de zonas de siniestros , etc...Por lo demas trabajaron muy bien, y seguramente no van a tener inconvenientes en avanzar apropiadamente. Saludos.

**Comment by Gona79** [comment]
Nota de la Semana 3: 10 (diez).

Chicos el objetivo del desarrollo del proyecto era conocer las herramientas de:
Creacion de proyectos ( Gantt, AGILE, Proyectos)
Ideacion, desarrollo y prueba. 
Pero principalmente el trabajo colaborativo.
Lo hicieron muy bien, adapten este trabajo en forma de plantilla y les sera util para el resto de los proyectos de la cursada. Continuen asi.

**Reply by tecnosisnet** [reply]
Gracias profe ! Seguiremos haciendo el mejor esfuerzo para finalizar la carrera. !

**Reply by evera8** [reply]
Gracias profe !!! Gracias compañeros. Seguimos adelante
